### PR TITLE
feat: show deposits and reclaims when present in transaction

### LINKED ISSUE #353: Deposits and refunds are not presented in the transaction summary
**Summary**

The Cardano explorer does not show deposits or refunds when say registering a staking key. So upon initial analysis of inputs and outputs, some transactions look unbalanced.

**Steps to reproduce**

Register a staking key and check the explorer

_Example transaction below:_

https://explorer.cardano-testnet.iohkdev.io/en/transaction?id=b2c2f15aa19192f87f14041a787e2534873553bc047bfbbf56b9a1c6cd69c021

**Expected results**

All key deposits should be shown on the blockchain

**Actual results**

At the moment there is no indication of any key deposit transaction or otherwise. Instead, we only see a balance increase.

**Conclusion**

The explorer should also report withdrawals key deposits

### PR DESCRIPTION
**Pull request by rhyslbw**
Shows Deposit or Deposit Reclaim in the `TransactionInfo` when present in
transaction

Closes #353

Screenshot from 2020-09-19 00-27-40
Screenshot from 2020-09-19 00-27-28
Screenshot from 2020-09-19 01-02-59

### CODE REVIEW

File: source/features/i18n/translations/de.json
Comment (rhyslbw) [inline source/features/i18n/translations/de.json:155]: @DominikGuzei This is just a Google Translate output, so let me know if it needs a tweak

File: source/features/i18n/translations/ja.json
Comment (rhyslbw) [inline source/features/i18n/translations/ja.json:155]: @darko-mijic This needs to be confirmed, is just Google Translate output

Comment (issoupadrink) [reply source/features/i18n/translations/ja.json:155]: Should confirm this with a translator as I can't see it in the English-Japanese dictionary: 
https://dictionary.cambridge.org/dictionary/english-japanese/deposit 
https://dictionary.cambridge.org/spellcheck/japanese-english/?q=%E9%A0%90%E3%82%8A%E9%87%91 
https://dictionary.cambridge.org/spellcheck/japanese-english/?q=%E9%A0%90%E9%87%91%E3%81%AE%E5%9B%9E%E5%8F%8E

**Review by issoupadrink — CHANGES_REQUESTED** [review]
We should pass this by a translator to make sure we get them right. I'll ask for someone to look at it :)

File: source/features/i18n/translations/ja.json
Comment (issoupadrink) [reply source/features/i18n/translations/ja.json:155]: Okay so I've confirmed with Junko and these should be: 

"deposit": "デポジット"
"depositReclaim": "デポジットの返還"

File: source/features/i18n/translations/de.json
Comment (issoupadrink) [reply source/features/i18n/translations/de.json:155]: I spoke to Tommy (CF) and he said these should be: 

"deposit": "Kaution", 
"depositReclaim": "Kautionsrückforderung"

**Review by DominikGuzei — APPROVED** [review]
The code looks good, let's wait for translation approval 👍

**Comment by rhyslbw** [comment]
@DominikGuzei Translations have been supplied, so we just need product approval now

**Review by darko-mijic — APPROVED** [review]

**Pull request merged**
